mutt-forms/widget-personnummer
# Feat: Required fields check

# Description

This PR adds the ability for the parent component to define which fields are required from the personal number lookup data. If that data is missing, then the widget will display an error and emit an event for the parent to handle.

Fixes #12 

## Type of change


# How Has This Been Tested?

Unit tests have been added.

To test in an app, you will need to `npm link` this package.

- Add `requiredFieldPaths` to the form schema SSN options
- Enter a protected identity number, e.g. 194203052222
- Click search
- The message should be displayed that the returned data is missing the required fields.


# Checklist:

- [x] Designer has reviewed the change
